org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -53,19 +53,18 @@ class State(ABC):
     def __ne__(self, obj):
         return type(self) != type(obj)
 
-    def handleEvent(self, eventName: str, vistedStates: set):
-        vistedStates.add(type(self._state))
+    def handleEvent(self):
         possible_transitions = []
         for k, v in self.transitions.items():
-            if k(eventName):
+            if k():
                 possible_transitions.append(v)
 
         if(possible_transitions):
             chosen_transition = random.randint(0, len(possible_transitions) - 1)
-            nextEvent = possible_transitions[chosen_transition]()
-            return nextEvent if (type(self._state) not in vistedStates) else ""
+            possible_transitions[chosen_transition]()
+            return ""
         elif(self._state != self):
-            return self._state.handleEvent(eventName, vistedStates)
```

Comment:
Does this function only return ""? If that's true maybe it shouldn't return anything?